resources/build/build-utils.sh

```diff
@@ -93,7 +93,9 @@ function findVersion() {
         fi
     fi
 
-    VERSION_WITH_TAG="$VERSION$VERSION_TAG"
+    # export these two so version.gradle can access them
+    export VERSION_ENVIRONMENT
+    export VERSION_WITH_TAG="$VERSION$VERSION_TAG"
```

Comment:
Instead of exporting environment, can we do the following in android/KMEA/build.sh etc? Does this work?

```shell
VERSION_ENVIRONMENT="$VERSION_ENVIRONMENT" VERSION_WITH_TAG="$VERSION_WITH_TAG" ./gradlew $DAEMON_FLAG clean $BUILD_FLAGS
```